webapp/src/components/Modals/SellModal/SellModal.container.ts

```diff
@@ -36,7 +38,8 @@ const mapState = (state: RootState): MapStateProps => {
     ),
     isAuthorizing:
       isLoadingType(getLoading(state), GRANT_TOKEN_REQUEST) ||
-      isLoadingType(getLoading(state), REVOKE_TOKEN_REQUEST)
+      isLoadingType(getLoading(state), REVOKE_TOKEN_REQUEST),
+    isLoadingCancel: isLoadingType(getLoading(state), CANCEL_ORDER_REQUEST)
```

Comment:
What do you think about renaming this to:
```suggestion
    isCancelling: isLoadingType(getLoading(state), CANCEL_ORDER_REQUEST)
```
it follows the same format as `isAuthorizing`.